public/video-ui/src/components/VideoSearch/VideoSearch.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+export default class VideoSearch extends React.Component {
+
+  constructor(props) {
```

Comment:
I don't think this is needed, I've taken out a few in my previous PR but there are still some hanging about.